book/content/research_compendia/research_compendia.md

```diff
@@ -120,7 +156,7 @@ To create a research compendium, follow these steps:
 
 ## Further reading
 
-The website https://research-compendium.science contains links to further ressources and publications on research compendia as well as links to examples.
+The website https://research-compendium.science contains links to further resources and publications on research compendia as well as links to examples.
 
```

Comment:
Can you see where is the line 125-128 is coming from?

Reply:
I am not sure I understand what you mean

Reply:
There were some placeholders in the file. I think it's ok to keep them in but 
I thought you might want to check.